### ISSUE TITLE
Error while building the generated connector code

### ISSUE THREAD

**Issue by girishramnani-crest** [hasura/ndc-sdk-go#47]
Commit: `9c7a7adc1cd36e4ab2e0da375f7fa70caecb27ab`

command: `ndc-sdk-go new -n ndc-sample -m github.com/girishramnani-crest/sample`

Ran `go build` on the generated code
got these error
```
./connector.generated.go:7:2: "encoding/json" imported and not used
./connector.generated.go:127:31: cannot use logger (variable of type zerolog.Logger) as *slog.Logger value in argument to connector_addSpanEvent
./connector.generated.go:138:2: logger declared and not used
```

**Comment by hgiasac** [comment]
You should download the binary from [v0.3.0](https://github.com/hasura/ndc-sdk-go/releases/tag/v0.3.0). The `main` branch is using Go 1.21 and there are breaking changes.

Or you can upgrade the dependency to main. The issue will be solved after we release v1.0

```sh
go get github.com/hasura/ndc-sdk-go@main
```

**Comment by hgiasac** [comment]
@girishramnani-crest would you mind trying again with version [v1.0.0](https://github.com/hasura/ndc-sdk-go/releases/tag/v1.0.0)? If there isn't any issue we can close this.
